build.gradle

```diff
@@ -59,9 +67,10 @@ sonarqube {
     }
 }
 
-jacocoTestReport {
-    reports {
-        xml.enabled true
+spotless {
```

Comment:
should we also add `groovyGradle` formatting?

Reply:
groovyGradle now added